# LTR-RT_k-mer_tool
A tool to explore LTR-retrotransposon dynamic across genomes with use of uniq k-mers

Mobile elements constitute a significant portion of plant genomes. Many contemporary approaches for their identification in raw genomic data rely on the partial assembly of short reads followed by taxonomic annotation. However, such methods may introduce biases due to the complexity of assembling repetitive sequences. In this study, we propose an alternative approach to investigate the mobilome using LTR retrotransposons (LTR-RT) in species of the genus _Avena_ L., leveraging high-quality genomic assemblies of multiple _Avena_ species.

LTR-RT search was conducted in the genome of _A. longiglumis_ Durieu and the subgenomes of _A. insularis_ Ladiz. using "LTR Finder" and "LTR retriver" tools. The obtained sequences of structurally intact LTR-RTs were annotated with "TE sorter". Subsequently, a more accurate clustering of LTR-RTs was performed through phylogenetic analysis based on amino acid gene sequences, and unique k-mers were identified for each cluster. Counting the occurrences of these k-mers in assembled genomes and raw genomic data available in the NCBI database for _Avena_ species allowed us to assess the representation of each cluster and draw conclusions about the dynamics of LTR-RTs within the genus. This work was supported by the Russian Science Foundation grant No. 22-24-01117.
